From 91a8dd6bff0c3965ab324b5c2a3c19c0a2931831 Mon Sep 17 00:00:00 2001
From: "lauren n. liberda" <lauren@selfisekai.rocks>
Date: Sat, 29 Jun 2024 00:26:40 +0000
Subject: [PATCH] unbundle: add missing absl_base header

Change-Id: Ie4edda0ec8661437652b2c4c324ac82137bfcfed
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5659519
Reviewed-by: Thomas Anderson <thomasanderson@chromium.org>
Reviewed-by: Lei Zhang <thestig@chromium.org>
Commit-Queue: Thomas Anderson <thomasanderson@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1321266}
---
 build/linux/unbundle/absl_base.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/linux/unbundle/absl_base.gn b/build/linux/unbundle/absl_base.gn
index 7da86e46147bdd..b5ce0863e6fde7 100644
--- a/build/linux/unbundle/absl_base.gn
+++ b/build/linux/unbundle/absl_base.gn
@@ -27,6 +27,7 @@ shim_headers("base_shim") {
   headers = [
     "call_once.h",
     "casts.h",
+    "dynamic_annotations.h",
   ]
 }
 
